html/upgrade.html.haml

```diff
@@ -0,0 +1,49 @@
+!!!
+%html
+  %head
+    %title
+      Machinery Error: incompatible system description
+    %meta{ :charset => 'utf-8' }
+    %link{ :href => "assets/machinery-base.css", :rel => "stylesheet", :type => "text/css" }
+    %link{ :href => "assets/machinery.css", :rel => "stylesheet", :type => "text/css" }
+    %script{ :src => "assets/jquery-2.1.1.min.js" }
+    %script{ :src => "assets/transition.js" }
+    %script{ :src => "assets/collapse.js" }
+    %script{ :src => "assets/modal.js" }
+    %script{ :src => "assets/jquery.searcher.min.js" }
+    %script{ :src => "assets/machinery-base.js" }
+    %script{ :src => "assets/show/machinery.js" }
+    %script{ :src => "assets/bootstrap-tooltip.js" }
+    %script{ :src => "assets/bootstrap-popover.js" }
+
+  %body
+    .container-fluid
+      #nav-bar
+        .row
+          .col-xs-1
+          .col-xs-10
+            %h1
+              System Description incompatible!
+        .row
+          .col-xs-1
+          .col-xs-4
+            Outdated format version of system description
```

Comment:
You are handling two cases, one in which case the description has an old format version and one which the format version is newer than the one machinery supports.

In the second case this message is not correct.